Week2/1.js

```diff
@@ -0,0 +1,5 @@
+let dutch = 'hoi';
+let turkish = 'merhaba';
+let spanish = 'hola';
+
+console.log('saying hello in different languages: ' + dutch, turkish, spanish);
```

Comment:
Do you think it's better to use `let` here, or `const`?

Reply:
it is better to use `const` in this case

Reply:
Yeah true, can you change it in the code?